### PR TITLE
Document Broker object in the managedClusterSet broker ns

### PR DESCRIPTION
**Pull request by sridhargaddam**
Submariner Globalnet feature will be supported in ACM 2.5
The required changes on the submariner-addon are already merged.
This PR updates the corresponding documentation on how to enable
the feature.

Related to: https://github.com/stolostron/backlog/issues/19403
Signed-off-by: Sridhar Gaddam <sgaddam@redhat.com>

### CODE REVIEW

File: services/deploy_submariner_api.adoc [hunk 1]
```diff
@@ -456,7 +475,7 @@ After the `ManagedClusterAddOn` is created, the `submariner-addon` deploys Subma
 +
 *Note:* The name of `ManagedClusterAddOn` must be `submariner`.
 
-. Repeat steps 2 and 3 for all of the managed clusters that you want to enable Submariner.
+. Repeat steps 3 and 4 for all of the managed clusters that you want to enable Submariner.
```
Comment (swopebe) [inline services/deploy_submariner_api.adoc:479]: spell out numbers and are we saying repeat the steps for all of the managed clusters that you want to enable Submariner _on?_

File: services/deploy_submariner_api.adoc [hunk 2]
```diff
@@ -418,6 +418,25 @@ Replace `managed-cluster-set-name` with a name for the `ManagedClusterSet` that
...
+apiVersion: submariner.io/v1alpha1
+kind: Broker
+metadata:
+     name: submariner-broker
+     namespace: <managed-cluster-set-name>-broker
+spec:
+     globalnetEnabled: false
+EOF
+----
+Replace `<managed-cluster-set-name>` with the name of the managed cluster. Also, to enable Submariner Globalnet in the ManagedClusterSet, configure the value of `globalnetEnabled` to true.
```
Comment (swopebe) [inline services/deploy_submariner_api.adoc:438]: code blocks here

File: services/deploy_submariner_api.adoc [hunk 3]
```diff
@@ -418,6 +418,25 @@ Replace `managed-cluster-set-name` with a name for the `ManagedClusterSet` that
 +
 After the `ManagedClusterSet` is created, the `submariner-addon` creates a namespace called `<managed-cluster-set-name>-broker` and deploys the Submariner broker to it.
 
+. Create the Broker Configuration on the hub cluster in the `<managed-cluster-set-name>-broker` namespace by entering the following command.
```
Comment (swopebe) [inline services/deploy_submariner_api.adoc]: Colon here

File: services/deploy_submariner_api.adoc (hunk 2 above)
Comment (sridhargaddam) [reply services/deploy_submariner_api.adoc:438]: Not sure i understand. I'm following the same convention/pattern used in the doc. Please see Line 415.
If you prefer that a code-block needs to be used, can you give an example?

**Review by sridhargaddam — COMMENTED** [review]
Thanks for reviewing @swopebe

**Comment by dfarrell07** [comment]
A slightly-modified version of this PR's commit is already merged to `2.5_stage`:

https://github.com/stolostron/rhacm-docs/commit/d4efa1bbb486ce2d7c5dd53d83da6bac20882673

```
--- a/services/submariner/deploy_submariner_api.adoc
+++ b/services/submariner/deploy_submariner_api.adoc
@@@ -408,7 -418,7 +408,11 @@@ Replace `managed-cluster-set-name` wit
  +
  After the `ManagedClusterSet` is created, the `submariner-addon` creates a namespace called `<managed-cluster-set-name>-broker` and deploys the Submariner broker to it.

++<<<<<<< HEAD:services/submariner/deploy_submariner_api.adoc
 +. Create the `Broker` configuration on the hub cluster in the `<managed-cluster-set-name>-broker` namespace by entering the following command:
++=======
+ . Create the Broker Configuration on the hub cluster in the `<managed-cluster-set-name>-broker` namespace by entering the following command:
++>>>>>>> fd29bb7e (Document Broker object in the managedClusterSet broker ns):services/deploy_submariner_api.adoc
  +
  ----
  cat << EOF | oc apply -f -
@@@ -425,10 -435,7 +429,14 @@@ spec
       globalnetEnabled: false
  EOF
  ----
++<<<<<<< HEAD:services/submariner/deploy_submariner_api.adoc
 ++
 +Replace `managed-cluster-set-name` with the name of the managed cluster.
 ++
 +Set the the value of `globalnetEnabled` to `true` if you want to enable Submariner Globalnet in the `ManagedClusterSet`.
++=======
+ Replace `<managed-cluster-set-name>` with the name of the managed cluster. Also, to enable Submariner Globalnet in the ManagedClusterSet, configure the value of `globalnetEnabled` to true.
++>>>>>>> fd29bb7e (Document Broker object in the managedClusterSet broker ns):services/deploy_submariner_api.adoc

  . Add one managed cluster to the `ManagedClusterSet` by entering the following command:
```

**Comment by dfarrell07** [comment]
I think this should be closed?

**Comment by sridhargaddam** [comment]
Yes @dfarrell07 the necessary changes are merged as part of a separate PR. So we can close this. Thanks.

**Pull request closed without merging**